test/api/unit/models/user_action.test.js

```diff
@@ -0,0 +1,70 @@
+const expect = require('chai').expect;
+const { UserAction } = require('../../../../api/models');
+
+const props = {
+  userId: 1,
+  actionId: 1,
+  targetId: 1,
+  targetType: 'site',
+  createdAt: new Date(),
+  updatedAt: new Date(),
+};
+
+describe('UserAction model', () => {
+  describe('instantiation', () => {
+    it('creates a new UserAction record', (done) => {
+      const model = UserAction.build(props);
+
+      model.validate().then(() => {
+        expect(model).to.exist;
+        done();
+      }).catch(done);
+    });
+  });
+
+  describe('validations', () => {
+    it('requires userId, targetId, targetType, actionId, and createdAt', (done) => {
+      const requiredFields = ['userId', 'targetId', 'targetType', 'actionId'];
```

Comment:
missing `createdAt` and `updatedAt`?

Reply:
those fields are automatically added by the ORM, so I don't think they need to be tested

Reply:
that's fine -- though you have `createdAt` listed up in the test case name :)

Reply:
ahh, my mistake!